spec/acceptance/errors_spec.rb

```diff
@@ -15,6 +14,7 @@
         expect(error).to be_a(Almodovar::HttpError)
         expect(error.message).to eq(error_message(code, resource_url))
         expect(error.response_body).to eq('<error>more info</error>')
+        expect(error.response_url).to eq(resource_url)  # Check for the response_url
```

Comment:
@Marc-Merino you've left some commented code in this file, was that on purpose? If so, I don't really understand them, could you clarify?

Reply:
They aren't necessary, I forgot to delete them.